SuperteamDAO/earn
# added global as a option

## What does this PR do?

This fixes #682 

## Where should the reviewer start?
You can start by checking footer section changes

## How should this be manually tested?

1. Navigate to the country selector and verify that "Global" is the default option.
2. Select any country from the dropdown.
3. Re-open the dropdown and confirm that "Global" appears as an option, allowing you to switch back to it.

## Any background context you want to provide?
Nope

## What are the relevant issues?
Fixes #682

## Screenshots (if appropriate)

before -
global was not a option once a country is selected

after-

<img width="1920" alt="Screenshot 2024-11-09 at 12 54 01 PM" src="https://github.com/user-attachments/assets/402944b6-e696-414d-96e9-c12e8cccc407">
